dev/data/structured-content-overrides.css

```diff
@@ -61,3 +61,16 @@
     display: none;
     /* remove-property background-color vertical-align width height margin-left background-color position */
 }
+.gloss-sc-ol,
+.gloss-sc-ul {
+    /* remove-property padding-left */
+}
+:root[data-glossary-layout-mode=compact] .gloss-sc-ul[data-sc-content=glossary] {
```

Comment:
Group these together for simplicity:
```css
:root[data-glossary-layout-mode=compact] .gloss-sc-ul[data-sc-content=glossary],
:root[data-glossary-layout-mode=compact] .gloss-sc-ul[data-sc-content=glossary] .gloss-sc-li,
:root[data-glossary-layout-mode=compact] .gloss-sc-ul[data-sc-content=glossary] .gloss-sc-li:not(:first-child)::before {
    /* remove-rule */
}
```

Reply:
Ha, that's actually what I tried first. But the tests do not like that. I think that only works if they are defined that way (as a group) in the `ext/css/structured-content.css` file. But the rules are all different there, so they can't be combined.

```
Running test-css-json.js...
Error: Could not find rule with matching selectors
    at generateRules (/home/stephen/Code/yomichan/dev/css-to-json-util.js:139:48)
    at main (/home/stephen/Code/yomichan/test/test-css-json.js:28:42)
    at testMain (/home/stephen/Code/yomichan/dev/util.js:127:15)
    at Object.<anonymous> (/home/stephen/Code/yomichan/test/test-css-json.js:35:5)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:827:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

Reply:
Hmm, I thought I handled that differently, but I guess not; my bad! (I should really be more familiar with my own code)